xc7/picosoc_demo/README.rst

```diff
@@ -65,6 +71,10 @@ The UART output should look as follows:
 
    PicoSoC uses baud rate of ``460800`` by default.
 
+.. note::
+
+   On the Nexys4DDR, the USB-UART does not work, so UART can be accessed from pins 1 and 2 of PMOD C.
```

Comment:
We should log an issue to figure out why the USB-UART is not working in the future.

Reply:
My hypothesis is that it is related to the baud rate of 460800. I have used the USB-UART on this board successfully with projects that used a baud rate of 115200

Reply:
I'm pretty sure that it is an issue related to the clock rate. I removed the BUFG that generates the clock and added a divider that slowed the entire design's clock by 4x, taking it from 100MHz to 25MHz. I don't know what clock rate the BUFG output is running at, but doing this allowed the onboard USB-UART to work at a baud rate of 57600

Reply:
I fixed the problem, it is working with the USB-UART.